e2e/nomostest/clean.go

```diff
@@ -601,8 +601,14 @@ func disableRootSyncDeletionPropagation(nt *NT) error {
 	return nil
 }
 
-func deleteTestAPIServices(nt *NT) error {
-	out, err := nt.Shell.Kubectl("delete", "apiservices", "-l", "testdata=true")
+func deleteAPIServices(nt *NT) error {
+	out, err := nt.Shell.Kubectl("delete", "apiservices", "-l", "testdata=true", "--ignore-not-found")
```

Comment:
Would be nice if this was using the delete and watch functions in the reset code. We would get better errors and it already handles both NotFound and the test label filter.